core/mutator.js

```diff
@@ -33,8 +33,8 @@ goog.require('Blockly.Events.Ui');
 goog.require('Blockly.Icon');
 goog.require('Blockly.utils');
 goog.require('Blockly.WorkspaceSvg');
-
-goog.require('goog.dom');
+goog.require('Blockly.Xml');
```

Comment:
Why do you need both of these requires?

Reply:
This file contains two calls to `Blockly.Xml.utils.createElement` and two calls to `Blockly.Xml.domToText`.  I could just require `Blockly.Xml` and rely on the (current) transitive dependency, but that's fragile.

Reply:
Okay.  Leaving it as-is sounds good to me.